### ISSUE TITLE
Add a shortcut for text search dialog

### ISSUE THREAD

**Issue by Maijin** [radareorg/cutter#1298]
**Is your feature request related to a problem? Please describe.**

A shortcut is missing for launching text search dialog

**Describe the solution you'd like**

Quick text search by doing Ctrl-F

**Comment by ITAYC0HEN** [comment]
text search where? in the disassembly? Only in the current view?

**Comment by Maijin** [comment]
Should be calling search on the current view

**Comment by ITAYC0HEN** [comment]
isn't there a search in most of our widgets already?
Specifically, in Graph and Disassembly we don't have.

Please be verbose and specific here.

**Comment by Maijin** [comment]
Yes for Disassembly and Graph - updated the ticket. That's from a legacy hackmd.
